dlib/dnn/loss.h

```diff
@@ -406,6 +406,14 @@ namespace dlib
         test_box_overlap overlaps_nms = test_box_overlap(0.4);
         test_box_overlap overlaps_ignore;
 
+        enum class use_image_pyramid : uint8_t
```

Comment:
Can you move it outside mmod_options?  Otherwise it's still a lot of typing.

Reply:
Right – fixed.